server/start_web.sh

```diff
@@ -2,11 +2,18 @@
 
 webserver_addr=$1
 
-python ./manage.py makemigrations collab
-python ./manage.py migrate
-has_admin=$(echo "select count(*) from auth_user where username='admin';" | python ./manage.py dbshell | sed 's/[^0-9]//g')
-if [ $has_admin -eq 0 ]; then
-    echo "Creating admin super user, please enter password"
-    python ./manage.py createsuperuser --username admin --email admin@local.com
+if [ -n "$IN_DOCKER" ]; then
```

Comment:
I think this is no longer needed, server is started as a service. I think we should revert this.